controllers/bsl.go

```diff
@@ -347,3 +352,15 @@ func (r *DPAReconciler) ensureBackupLocationHasVeleroOrCloudStorage(dpa *oadpv1a
 	}
 	return nil
 }
+
+func (r *DPAReconciler) resticCustomCAEnabled(bslSpec velerov1.BackupStorageLocationSpec, dpa *oadpv1alpha1.DataProtectionApplication) (bool, error) {
+	insecureSkipTLSVerify, skipPresent := bslSpec.Config["insecureSkipTLSVerify"]
+	if !skipPresent || insecureSkipTLSVerify != "false" {
```

Comment:
Why are we returning `false` here ?

Reply:
if insecureSkipTlsVerify is true or does not exists, then customCA is not used for communication.